README.md

```diff
@@ -28,9 +28,9 @@ This is a plugin for Homebridge, allowing communication with Alarm.com endpoints
 - Locks
   - Lock/Unlock switch
 - Garage Doors
-  - Open/Close switch 
-- Thermostats 
-  - Set mode Off/Heat/Cool/Auto 
+  - Open/Close switch
+- Thermostats
+  - Set mode Off/Heat/Cool/Auto
```

Comment:
Could you remove this file from your PR? It rather not have whitespace changes in the merge. Thanks!